### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

**Comment by eerison** [comment]
@VincentLanglet I have created this PR: https://github.com/sonata-project/SonataPageBundle/pull/1571

just to help you with the page change for layout!

after you merge this, you can close this #1571

**Comment by eerison** [comment]
Please apply those 2 changes

https://github.com/sonata-project/SonataPageBundle/pull/1571/files#diff-f0461df8f4a53195f2b565ad37746df1e8076ff748edb18c3e2689581f2db4caR86

https://github.com/sonata-project/SonataPageBundle/pull/1571/files#diff-99821531fe8569e4745708ce6f87e337c82557391aaa9927f3c737260fae004bR22

File: src/Controller/PageAdminController.php [hunk 1]
```diff
@@ -237,37 +257,50 @@ public function composeAction(Request $request): Response
...
 
-                if (null !== $container) {
-                    foreach ($blockServices as $code => $service) {
-                        if (\in_array($code, $container['blocks'], true)) {
-                            continue;
-                        }
+            $template = $templateManager->get($page->getTemplateCode());
+            if (null === $template) {
+                throw new NotFoundHttpException(
+                    sprintf('unable to find the template with code : %s', $templateCode)
```
Comment (jordisala1991) [inline src/Controller/PageAdminController.php]:
```suggestion
                    sprintf('Unable to find the template with code : %s', $templateCode)
```

Comment (VincentLanglet) [reply src/Controller/PageAdminController.php]: Done

File: src/Block/BreadcrumbBlockService.php [hunk 2]
```diff
@@ -83,7 +83,7 @@ protected function getMenu(BlockContextInterface $blockContext): ItemInterface
 
     private function getName(): string
     {
-        return 'sonata.page.block.breadcrumb';
+        return 'page';
```
Comment (jordisala1991) [inline src/Block/BreadcrumbBlockService.php]: IMO this is not the correct change. The metadata can return the previous value and the handleContext can check against page, and we can remove this method.

Comment (VincentLanglet) [reply src/Block/BreadcrumbBlockService.php]: I reverted.

I didn't follow this dicussion so prefer to let @eerison doing another PR about it.

**Review by jordisala1991 — APPROVED** [review]

File: src/Block/BreadcrumbBlockService.php (hunk 2 above)
Comment (eerison) [reply src/Block/BreadcrumbBlockService.php]: well but this values need to be equals passed by template!

if we use something different of `page`, then we still have the same problem to use `sonata.page.block.breadcrumb`

what is your suggestion for this @jordisala1991 ?

**Pull request merged**